### ISSUE TITLE
Ability to merge AST?

### ISSUE THREAD

**Issue by mgcrea** [Urigo/merge-graphql-schemas#180]
I'm looking for a way to merge ASTs of graphql schemas:

I'm using webpack on the server side with the following rule:

```
      {
        test: /\.(graphql)$/,
        loader: 'graphql-tag/loader'
      }
```
So this gives me parsed AST instead of raw strings when I import the `.graphql` files.

Would be great if this module could implement a lower level function to merge already parsed ASTs.

**Comment by kamilkisiela** [comment]
#196 makes `mergeTypeDefs` accept DocumentNode, String, GraphQLSchema, Source and pretty much all you can imagine

**Comment by kamilkisiela** [comment]
It's available now under `v1.6.0-beta.2`. Let us know if it works :)
